scripts/copyrightLinter.js

```diff
@@ -22,10 +36,10 @@ const copyrightLine2 = `See LICENSE.md in the project root for license terms and
 const copyrightBannerScss = `// ${copyrightLine1}\n// ${copyrightLine2}`;
 const copyrightBannerHtml = `<!--\n  ${copyrightLine1}\n  ${copyrightLine2}\n-->`;
 const copyrightBannerJs =
-  '/*---------------------------------------------------------------------------------------------\n ' +
-  `* ${copyrightLine1}\n ` +
-  `* ${copyrightLine2}\n ` +
-  '*--------------------------------------------------------------------------------------------*/';
+  '/*---------------------------------------------------------------------------------------------\n' +
+  ` * ${copyrightLine1}\n` +
+  ` * ${copyrightLine2}\n` +
+  ' *--------------------------------------------------------------------------------------------*/';
```

Comment:
Are these changes intended?

Reply:
yes they are. i moved the whitespace around to be more readable. the end result is still the same